api/app/routers/hfi_calc.py

```diff
@@ -352,7 +357,7 @@ async def get_fire_centres(response: Response):
         logger.info('/hfi-calc/fire-centres')
         # we can safely cache the fire centres, as they don't change them very often.
         # the eco-division logic is very slow, and chomps up 2 seconds!
-        response.headers["Cache-Control"] = "max-age=86400"
+        response.headers["Cache-Control"] = "max-age=0"
         fire_centres_list = await hydrate_fire_centres()
         return HFIWeatherStationsResponse(fire_centres=fire_centres_list)
```

Comment:
I would add something here - that caches the response in redis if it doesn't exist.